File: 9ariniPi-Symfony/src/Entity/Reclamation.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use phpDocumentor\Reflection\Types\Boolean;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * Reclamation
 *
 * @ORM\Table(name="reclamation", indexes={@ORM\Index(name="FOREIGN 1", columns={"utilisateurID"})})
 * @ORM\Entity(repositoryClass="App\Repository\ReclamationRepository")
 */
class Reclamation
{
    /**
     * @var int
     *
     * @ORM\Column(name="reclamationID", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $reclamationid;

    /**
     * @var string
     *
     * @ORM\Column(name="categorieReclamation", type="string", length=9, nullable=false)
     */
    private $categoriereclamation;

    /**
     * @var string
     *
     * @ORM\Column(name="messageReclamation", type="string", length=255, nullable=false)
     */
    private $messagereclamation;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateReclamation", type="datetime", nullable=false)
     */
    private $datereclamation;

    /**
     * @var string
     *
     * @ORM\Column(name="reponseReclamation", type="string", length=255, nullable=true)
     */
    private $reponsereclamation;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dateReponse", type="datetime", nullable=true)
     */
    private $datereponse;

    /**
     * @var string
     *
     * @ORM\Column(name="etatReclamation", type="string", length=8, nullable=false)
     */
    private $etatreclamation;

    /**
     * @var bool
     *
     * @ORM\Column(name="etatReponse", type="boolean", nullable=true)
     */
    private $etatreponse;

    /**
     * @var \Utilisateur
     *
     * @ORM\ManyToOne(targetEntity="Utilisateur")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="utilisateurID", referencedColumnName="id")
     * })
     */
    private $utilisateurid;

    public function getReclamationid(): ?int
    {
        return $this->reclamationid;
    }

    public function getMessagereclamation(): ?string
    {
        return $this->messagereclamation;
    }

    public function setMessagereclamation(string $messagereclamation): self
    {
        $this->messagereclamation = $messagereclamation;

        return $this;
    }

    public function getReponsereclamation(): ?string
    {
        return $this->reponsereclamation;
    }

    public function setReponsereclamation(?string $reponsereclamation): self
    {
        $this->reponsereclamation = $reponsereclamation;

        return $this;
    }

    public function getCategoriereclamation(): ?string
    {
        return $this->categoriereclamation;
    }

    public function setCategoriereclamation(string $categoriereclamation): self
    {
        $this->categoriereclamation = $categoriereclamation;

        return $this;
    }

    public function getDatereclamation(): ?\DateTimeInterface
    {
        return $this->datereclamation;
    }

    public function setDatereclamation(\DateTimeInterface $datereclamation): self
    {
        $this->datereclamation = $datereclamation;

        return $this;
    }

    public function getDatereponse(): ?\DateTimeInterface
    {
        return $this->datereponse;
    }

    public function setDatereponse(\DateTimeInterface $datereponse): self
    {
        $this->datereponse = $datereponse;

        return $this;
    }

    public function getEtatreclamation(): ?string
    {
        return $this->etatreclamation;
    }

    public function setEtatreclamation(string $etatreclamation): self
    {
        $this->etatreclamation = $etatreclamation;

        return $this;
    }

    public function getEtatreponse(): ?bool
    {
        return $this->etatreponse;
    }

    public function setEtatreponse(bool $etatreponse): self
    {
        $this->etatreponse = $etatreponse;

        return $this;
    }

    public function getUtilisateurid(): ?Utilisateur
    {
        return $this->utilisateurid;
    }

    public function setUtilisateurid(Utilisateur $utilisateurid): self
    {
        $this->utilisateurid = $utilisateurid;

        return $this;
    }


}

File: 9ariniPi/src/entities/Reclamation.java
package entities;

import java.sql.Date;

/**
 *
 * @author yahia
 */
public class Reclamation {

    private int reclamationID, utilisateurID, formateurid, formationid;
    private String categorieReclamation, messageReclamation, reponseReclamation, etatReclamation;
    private Date dateReclamation, dateReponse;
    private boolean etatReponse;

    
    public Reclamation() {
    }
    
    public Reclamation(int utilisateurID, String categorieReclamation, String messageReclamation, String reponseReclamation, String etatReclamation, Date dateReclamation, Date dateReponse, boolean etatReponse) {
        this.utilisateurID = utilisateurID;
        this.categorieReclamation = categorieReclamation;
        this.messageReclamation = messageReclamation;
        this.reponseReclamation = reponseReclamation;
        this.etatReclamation = etatReclamation;
        this.dateReclamation = dateReclamation;
        this.dateReponse = dateReponse;
        this.etatReponse = etatReponse;
    }

    public Reclamation(int utilisateurID, int formateurid, int formationid, String categorieReclamation, String messageReclamation, String reponseReclamation, String etatReclamation, Date dateReclamation, Date dateReponse, boolean etatReponse) {
        this.utilisateurID = utilisateurID;
        this.formateurid = formateurid;
        this.formationid = formationid;
        this.categorieReclamation = categorieReclamation;
        this.messageReclamation = messageReclamation;
        this.reponseReclamation = reponseReclamation;
        this.etatReclamation = etatReclamation;
        this.dateReclamation = dateReclamation;
        this.dateReponse = dateReponse;
        this.etatReponse = etatReponse;
    }
    
    public Reclamation(int reclamationID, int utilisateurID, int formateurid, int formationid, String categorieReclamation, String messageReclamation, String reponseReclamation, String etatReclamation, Date dateReclamation, Date dateReponse, boolean etatReponse) {
        this.reclamationID = reclamationID;
        this.utilisateurID = utilisateurID;
        this.formateurid = formateurid;
        this.formationid = formationid;
        this.categorieReclamation = categorieReclamation;
        this.messageReclamation = messageReclamation;
        this.reponseReclamation = reponseReclamation;
        this.etatReclamation = etatReclamation;
        this.dateReclamation = dateReclamation;
        this.dateReponse = dateReponse;
        this.etatReponse = etatReponse;
    }
    
    public static Date getCurrentDate() {
        java.util.Date date = new java.util.Date();
        java.sql.Date sqlDate = new java.sql.Date(date.getTime());
        return sqlDate;
    }

    public int getReclamationID() {
        return reclamationID;
    }

    public void setReclamationID(int reclamationID) {
        this.reclamationID = reclamationID;
    }

    public int getUtilisateurID() {
        return utilisateurID;
    }

    public void setUtilisateurID(int utilisateurID) {
        this.utilisateurID = utilisateurID;
    }

    public int getFormateurid() {
        return formateurid;
    }

    public void setFormateurid(int formateurid) {
        this.formateurid = formateurid;
    }

    public int getFormationid() {
        return formationid;
    }

    public void setFormationid(int formationid) {
        this.formationid = formationid;
    }

    public String getCategorieReclamation() {
        return categorieReclamation;
    }

    public void setCategorieReclamation(String categorieReclamation) {
        this.categorieReclamation = categorieReclamation;
    }

    public String getMessageReclamation() {
        return messageReclamation;
    }

    public void setMessageReclamation(String messageReclamation) {
        this.messageReclamation = messageReclamation;
    }

    public String getReponseReclamation() {
        return reponseReclamation;
    }

    public void setReponseReclamation(String reponseReclamation) {
        this.reponseReclamation = reponseReclamation;
    }

    public String getEtatReclamation() {
        return etatReclamation;
    }

    public void setEtatReclamation(String etatReclamation) {
        this.etatReclamation = etatReclamation;
    }

    public Date getDateReclamation() {
        return dateReclamation;
    }

    public void setDateReclamation(Date dateReclamation) {
        this.dateReclamation = dateReclamation;
    }

    public Date getDateReponse() {
        return dateReponse;
    }

    public void setDateReponse(Date dateReponse) {
        this.dateReponse = dateReponse;
    }

    public boolean isEtatReponse() {
        return etatReponse;
    }

    public void setEtatReponse(boolean etatReponse) {
        this.etatReponse = etatReponse;
    }
    
    @Override
    public String toString() {
        return "Reclamation{" + "reclamationID=" + reclamationID + ", utilisateurID=" + utilisateurID + ", formateurid=" + formateurid + ", formationid=" + formationid + ", categorieReclamation=" + categorieReclamation + ", messageReclamation=" + messageReclamation + ", reponseReclamation=" + reponseReclamation + ", etatReclamation=" + etatReclamation + ", dateReclamation=" + dateReclamation + ", dateReponse=" + dateReponse + ", etatReponse=" + etatReponse + '}';
    }

    
    
}
